public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js

```diff
@@ -172,20 +175,38 @@ class DefineMonitor extends Component {
     // 1. The actual query that will be saved on the monitor, to get accurate query performance stats
     // 2. The UI generated query that gets [BUCKET_COUNT] times the aggregated buckets to show past history of query
     // If the query is an extraction query, we can use the same query for results and query performance
```

Comment:
Please remove these comments, seems they have been moved to L185-188.

Reply:
Removed.

Reply:
Thank you.